api/operator/v1alpha1/authentication_types.go

```diff
@@ -238,6 +230,8 @@ func NewMigrationInProgressCondition() *metav1.Condition {
 	}
 }
 
+// Creates a new ConditionMigrationsRunning condition for when the migration Job
+// has stopped running.
 func NewMigrationFinishedCondition() *metav1.Condition {
 	return &metav1.Condition{
 		Type:    ConditionMigrationsRunning,
```

Comment:
Hi @rwhundley , you meant  `ConditionMigrationsFinished` here?

Reply:
The reason this distinction was made was because, as a single condition, the window of time where one could see that a migration had failed was very small before it would be updated to state that migrations were in progress. Tracking as two separate conditions allows for it to be possible to see from the Authentication CR that migrations have been failing repeatedly and whether or not migrations are actively running or not without as much of a timing constraint.